esi_leap/api/controllers/v1/utils.py

```diff
@@ -122,8 +122,8 @@ def check_lease_policy_and_retrieve(request, policy_name, lease_ident,
 
 
 def check_offer_policy_and_retrieve(request, policy_name, offer_ident,
-                                    status_filter=None):
-    offer = get_offer(offer_ident, status_filter)
+                                    status_filters=[]):
+    offer = get_offer(offer_ident, status_filters)
```

Comment:
The 'get_offer' method expects status_filter to be a string, so this will cause issues. You'll have to modify get_offer, and likely offer_get_all in db/sqlalchemy/api.py